### DISCUSSION TITLE
Rows Splitting in multiple lines

### DISCUSSION THREAD

**Discussion by souravgajbhiye** [jsvine/pdfplumber#510] [Q&A]
How can we change the background color of the document to white and then use extract_table() .
For some documents it has been observed that rows with non-white background color gets split into multiple lines.

**Comment by samkit-jain** [comment]
Hi @souravgajbhiye Appreciate your interest in the library. Could you please provide more details as per https://github.com/jsvine/pdfplumber/blob/stable/.github/ISSUE_TEMPLATE/bug-report.md to help you better?

Or perhaps your question is a duplicate of https://github.com/jsvine/pdfplumber/issues/488?

**Reply by souravgajbhiye** [reply]
I will not be able to share the pdf file. But will try to explain the issue. So, the PDF file consist of rows which has some different color in it also pdfplumber is able to detect hidden lines in those rows which are not required (i tried the solution that you shared for #488 but it didn't work for me non_stroking_color is not 0 its 0.8 in my case) .

**Reply by samkit-jain** [reply]
Does it work if you change the value from 0 to 0.8?

**Reply by souravgajbhiye** [reply]
Sadly, it does not , it still misses the lines of the rows which has some color in it.

**Reply by samkit-jain** [reply]
In that case, I would recommend you to find the properties that are unique to those unwanted lines that you can use to filter out them and then run the table extraction.

**Reply by souravgajbhiye** [reply]
[output_objects.txt](https://github.com/jsvine/pdfplumber/files/7254087/output_objects.txt)   <- contains result of pdf.pages[0].objects
[1_updated_augp.pdf](https://github.com/jsvine/pdfplumber/files/7254089/1_updated_augp.pdf)  <- is the layout of one table from many tables present in a single page. If you look at the rows these are consecutively colored as white and light blue. As you can see Row 3 has a larger row height as compared to the others, col5 of Row3 contains 3 lines of text written one below other, which when extracted using the extract_tables() results into 3 lists(i.e lines) where as it should be collated in one line as its only one row (i.e Row 3).
It would be of great help ,if you can help in resolving this issue .
Thank you so much for your prompt responses.
